cypress/e2e/fundingFlow.cy.ts

```diff
@@ -0,0 +1,29 @@
+import { geyserUrl } from '../contants'
+import { onChainRefundFlow, onChainSuccessFlow, testLightningSuccessFlow } from './funding'
+
+describe('Testing Project with lightning node', () => {
+  beforeEach(() => {
+    cy.visit(`${geyserUrl}/project/lndtestproject`, {
```

Comment:
What do you think about setting the project name as a constant? That way if it changes in the future, we can just update it in one place.